### ISSUE TITLE
Add tab pause/play control

### ISSUE THREAD

**Issue by PalmerAL** [minbrowser/min#704]

**Comment by shalva97** [comment]
Yes. and they should be in Task View, next to close button. Also with Next and Previous buttons. That would be awesome!

something like this:
![image](https://user-images.githubusercontent.com/22417494/93190560-90fc7880-f754-11ea-8d1e-3fa85f76347b.png)
